tests/sharing/scalability/base_tests.rb

```diff
@@ -0,0 +1,84 @@
+require_relative '../boot'
+
+$stdout.sync = true
+
+def create_instances(names)
+  insts = Array.new(names.length)
+  names.each_with_index do |name, i|
+    insts[i] = Instance.create name: name
+  end
+  insts
+end
+
+def create_sharing(insts, obj)
+  inst_sharer = insts[0]
+  sharing = Sharing.new
+  sharing.rules << Rule.sync(obj)
+  sharing.members << inst_sharer
+  insts.each_with_index do |inst, i|
+    if i != 0
+      contact = Contact.create inst_sharer, givenName: inst.name
+      sharing.members << contact
+    end
+  end
+  inst_sharer.register sharing
+  sleep 1
+  insts.each_with_index { |inst, i| inst.accept sharing if i != 0 }
+  sharing
+end
+
+def create_files(inst, n_files, dir_id)
+  print "Create #{n_files} files... "
+  files = Array.new(n_files)
+  n_files.times do |i|
+    file_name = "#{Faker::Internet.unique.slug}.txt"
+    files[i] = CozyFile.create inst, dir_id: dir_id, name: file_name
+  end
+  print "Done.\n"
```

Comment:
In Ruby, it's more common to us `puts` than `print`. Cf https://www.garethrees.co.uk/2013/05/04/p-vs-puts-vs-print-in-ruby/